src/components/pages/WordCloud/WordCloud.js

```diff
@@ -1,89 +1,88 @@
-import React from "react";
-import useWindowDimensions from "./windowdimensions";
-import "./wordcloud.css";
-import axios from "axios";
+import React from 'react';
+import useWindowDimensions from './windowdimensions';
+import './wordcloud.css';
+import axios from 'axios';
 
 export default function WordCloud(props) {
```

Comment:
how is this file related to this Trello card? https://trello.com/c/l4R6fQJW/473-update-the-gallery-reducer-to-receive-the-data-from-the-new-schema 
Is this PR atomic?

Reply:
I restored the file from main so it has all the changes recently made on main.